frontend/libs/studio-components/src/components/StudioAvatar/StudioAvatar.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { type ImgHTMLAttributes, type ReactElement } from 'react';
+import classes from './StudioAvatar.module.css';
+import cn from 'classnames';
+import { PersonCircleIcon } from '@studio/icons';
+
+export type StudioAvatarProps = {
+  avatarElement?: ImgHTMLAttributes<HTMLImageElement>;
+};
+
+export const StudioAvatar = ({ avatarElement }: StudioAvatarProps): ReactElement => {
+  if (!avatarElement) {
+    return <PersonCircleIcon className={classes.avatar} />;
+  }
+  return <img {...avatarElement} className={cn(avatarElement.className, classes.avatar)} />;
+};
```

Comment:
Bare et forslag her. Hva med å sette `StudioAvatarProps` til å være lik `ImgHTMLAttributes<HTMLImageElement>`?
```suggestion
export type StudioAvatarProps = ImgHTMLAttributes<HTMLImageElement>;

export const StudioAvatar = ({ src, className: givenClass, ...rest }: StudioAvatarProps): ReactElement => {
  const className = cn(givenClass, classes.avatar);
  return src
    ? <img src={src} className={className} {...rest} />
    : <PersonCircleIcon className={className} />;
};
```

Reply:
I totally agree! 😄